lib/Parse/ParseExpr.cpp

```diff
@@ -1386,6 +1386,55 @@ ExprResult Parser::ParseCastExpression(bool isUnaryExpression,
   // are compiling for OpenCL, we need to return an error as this implies
   // that the address of the function is being taken, which is illegal in CL.
 
+  // With primary-expression, before parsing postfix-expression, check to make
+  // sure that Expr is declRefExpr of generic function with _For_any specifier
+  Expr* resExpr = Res.get();
```

Comment:
Please add spaces put the '*'* next to the variable name: `Expr *resExpr`, not `Expr* resExpr`.